File: Python/baekjoon/단계별로 풀어보기/정수론 및 조합론/5086_배수와 약수.py
# https://www.acmicpc.net/problem/5086

from sys import stdin

input = stdin.readline
while True:
    a, b = map(int, input().split())
    if a == b == 0:
        break
    if b // a == b / a:
        print('factor')
    elif a // b == a / b:
        print('multiple')
    else:
        print('neither')

File: C#/Baekjoon/약수, 배수와 소수/5086_배수와 약수.cs
//https://www.acmicpc.net/problem/5086

while (true)
{
    int[] n = Array.ConvertAll(Console.ReadLine().Split(), int.Parse);
    if (n[0] == 0) break;
    if (n[0] % n[1] == 0) Console.WriteLine("multiple");
    else if (n[1] % n[0] == 0) Console.WriteLine("factor");
    else Console.WriteLine("neither");
}